model/regenerate.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -
+
+mvn \
+validate \
+org.codehaus.mojo:sql-maven-plugin:execute \
+org.flywaydb:flyway-maven-plugin:migrate \
+org.jooq:jooq-codegen-maven:generate
```

Comment:
This is our new script which can be manually invoked to re-generate the Java-to-database glue code. I believe validate is necessary because without it Maven will not attempt to ensure that all declared dependencies (including the build plugins themselves) are available locally in the case where you're invoking a build plugin directly.

Reply:
would be nice if we a) added to README to explain that this should be run occasionally (e.q. when we bump jooq and/or change schema) and b) added a step in CI where we attempt to run the code generation and ensure there's no file diff.

Reply:
Could it be a test? Have the test regen the files and diff. If the test fails, it prints instructions for how to fix.

Reply:
Are you referring to Travis CI or our internal build CI? That might be doable, assuming we use a similar setup to the server sub-project where we're using a profile.